.github/workflows/unit-test-status-check.yml

```
@@ -61,11 +70,14 @@ jobs:
       run_branch_validation_shared_tests,
       run_script_generator_tests,
       get_sdk_version]
-    uses: KinsonDigital/Infrastructure/.github/workflows/run-csharp-tests.yml@v11.0.0
+    uses: KinsonDigital/Infrastructure/.github/workflows/run-csharp-tests.yml@v13.1.0
     with:
       project-name: "${{ vars.PROJECT_NAME }}IntegrationTests"
       build-config: Debug
       net-sdk-version: "${{ needs.get_sdk_version.outputs.sdk-version }}"
+      checkout-ref: ${{ github.event.pull_request.head.ref }}
+      checkout-repository: ${{ github.event.pull_request.head.repo.full_name }}
+
```

Remove the extra empty line that was just added.

The standard layout of jobs across the entire organization is 2 empty lines between jobs.
This is not required, but just a YAML/workflow coding standard that is enforced to maintain readability.

```suggestion
      checkout-ref: ${{ github.event.pull_request.head.ref }}
      checkout-repository: ${{ github.event.pull_request.head.repo.full_name }}
```